### PR TITLE
add scala no trait

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: server/app/service/ScalaTestRunner.scala [hunk 1]
```diff
@@ -62,6 +62,20 @@ object ScalaTestRunner {
     }
   }
 
+  def execSuiteNoTrait(solution: String, suite: String, executor: (=> String) => String = tryExec): String = {
```
Comment (novakov-alexey) [inline server/app/service/ScalaTestRunner.scala]: >> executor: (=> String) => String = tryExec what is it?

**Comment by novakov-alexey** [comment]
Everything is fine, except one place.

File: server/app/service/ScalaTestRunner.scala (hunk 1 above)
Comment (novakov-alexey) [inline server/app/service/ScalaTestRunner.scala]: Scaladoc would be great to see for this API method.

Comment (DmytroOrlov) [inline server/app/service/ScalaTestRunner.scala]: Can we talk to discuss it?

**Comment by DmytroOrlov** [comment]
Thank you!

**Pull request merged**